examples/common-chunk-grandchildren/build.js

```diff
@@ -0,0 +1 @@
+require("../build-common");
```

Comment:
add `global.NO_TARGET_ARGS = true;` before this line to disable the default and allow multiple configurations.